sound/soc/bcm/allo-piano-dac-plus.c

```diff
@@ -158,7 +153,7 @@ static int __snd_allo_piano_dsp_program(struct snd_soc_pcm_runtime *rtd,
 		} else { /* low */
 			sprintf(firmware_name,
```

Comment:
Sorry, should have noticed this first time around.
snprintf(firmware_name, sizeof(firmware_name), ....) to avoid any potential for buffer overflow.